client/components/App.jsx

```diff
@@ -14,7 +14,7 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    fetch('/api/seller/5da6350a92b86a1332313b4c')
```

Comment:
You may want to consider adding a productId in addition to the one that is automatically generated by mongodb so that you can do look up by simpler numbers. Not necessary but would make swapping between displaying different db entries easier